lib/engine/devices/builtins/thingengine.builtin.ts

```diff
@@ -75,6 +75,10 @@ export default class MiscellaneousDevice extends Tp.BaseDevice {
         return [{ random: Math.round(low + (Math.random() * (high! - low))) }];
     }
 
+    get_get_timezone() {
+        return [{ timezone: new Tp.Value.Entity(Temporal.Now.timeZone().id, Temporal.Now.timeZone().toString()) }];
```

Comment:
This is not correct. The timezone of the user is `this.platform.timezone`. You're returning the system timezone which is not the right one in the cloud.

By the way, the reason why users ask "what timezone are you in" is because we had bugs where the user's timezone would not be respected. Those bugs have been fixed so I think this question will be less relevant.